api/resources_portal/test/views/__init__.py

```diff
@@ -0,0 +1,4 @@
+from resources_portal.test.views.test_material_request import (
+    TestMaterialRequestListTestCase,
+    TestSingleMaterialRequestTestCase,
+)
```

Comment:
Why is this necessary?

Reply:
Ah- this was my mistake. I had it here so I could run these tests specifically instead of all of them. I'll remove.

Reply:
Ohhhh. You couldn't run just them without it?

Reply:
I don't think so. In order to run specific tests, you need to make the module which contains the tests visible to the run_tests.sh script. At some point I'll make a PR which adds all the tests to the __init__.py so the script can see them, but it's out of scope for this.